turk-one/app/Views-Controllers/Admin/admin-workers.html

```diff
@@ -190,11 +193,11 @@ <h6 class="bottom-buffer-25">References available</h6>
 
 				<p><span class="text-muted">explanation: </span> {{explain}}</p>
 
-				<p><span class="text-muted">time spent:</span> {{timeInMinutes}}m</p>
+				<p><span class="text-muted">time spent:</span> {{_time}}</p>
```

Comment:
Oh! I see what you're saying. How about something like "[time name here]Formatted"? That way we keep camel case and "formatted" denotes that this is not the exact representation in our data, but a formatted output of that data